From adc94393dbeddf9e027d1b2dfff7c1bab975224e Mon Sep 17 00:00:00 2001
From: Dmitry Kazakov <dimula73@gmail.com>
Date: Wed, 20 Dec 2023 15:03:15 +0100
Subject: [PATCH] Fix finding libspng with CMake (#262)

The macro should be included before usage, this include is not
auto-generated by @PACKAGE_INIT@
---
 cmake/Config.cmake.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/cmake/Config.cmake.in b/cmake/Config.cmake.in
index d0123db..42a0b78 100644
--- a/cmake/Config.cmake.in
+++ b/cmake/Config.cmake.in
@@ -1,5 +1,7 @@
 @PACKAGE_INIT@
 
+include(CMakeFindDependencyMacro)
+
 find_dependency(ZLIB REQUIRED)
 
 include("${CMAKE_CURRENT_LIST_DIR}/@targets_export_name@.cmake")
